### PR TITLE
Bugfix/ask email confirmation when adding organisation manager

### LINKED ISSUE #3346: Ask email confirmation when adding organisation manager.
# Task
1. After a manager added a new manager, email is sent.
2. When request is sent, it shows manager as a pending manager as in picture.
3. After email is confirmed, show as normal (without any label).
4. Change text to 
`An organization manager has the same rights as the original API manager to edit and delete the APIs which are connected to this organization.`

## Mock-up
untitled

## Resources
Label (use default one): https://getbootstrap.com/docs/3.3/components/#available-variations

### PR DESCRIPTION
**Pull request by deligence-dharmendra**
Closes #3346

## Changes

- changes in language, schema, organization router file
- add new send email verification and verify email method in organization method

## Developer checklist
This checklist is to be completed by the PR developer:

- [ ] Alternative solutions were compared/discussed before writing code
    - [ ] trade-offs with this solution are considered acceptable
- [ ] Code in this PR adheres to the [project styleguide](CONTRIBUTING.md)
- [ ] This pull request does not decrease project test coverage
- [ ] If the code changes existing database collection(s), migration has been written
- [ ] If UI texts are added or changed, all texts are internationalized

## Reviewer checklist
Reviewed by: @username1

This list is to be completed by the pull request reviewer:
- [ ] Code works as described/expected
- [ ] Code seems to be error free
    - [ ] no browser console errors visible
    - [ ] no server console errors visible
    - [ ] passes CI build
- [ ] Code is written in a way that promotes maintainability
    - [ ] easy to understand
    - [ ] well organized
    - [ ] follows project coding standards and conventions
    - [ ] well documented

### CODE REVIEW

**Comment by Nazarah** [comment]
Fix the following:
"In case mail sending fails, the s-alert notifying the user about this remains in the page.
It doesn't automatically vanishes."

**Comment by Nazarah** [comment]
@deligence-dharmendra @matleppa I was trying to test the fix from my localhost.
I tried to setup the email configuration from the settings and tried to add another user as a manager.
On clicking Add Manager, it shows the message "Sending Verification Email failed" as S alert.
My question is that do I need to set up valid email configuration information (i.e. same as that in nightly APInf) in order to verify the mail settings?

**Comment by deligence-dharmendra** [comment]
@Nazarah Yes, you need to set up valid email configuration information. Otherwise, email id does not add to the organization and its show error message as an s-alert on the same page.

sending email verification error

**Comment by Nazarah** [comment]
@deligence-dharmendra @matleppa Just ran the feature in my localhost.
When user confirms the email by clicking on the link, he gets redirected to the home page of APInf.
Shouldn't it be the case that on clicking the verification link, user gets redirected to the Organization profile page?

File: apinf_packages/organizations/client/add/autoform.js [hunk 1]
```diff
@@ -22,6 +22,12 @@ AutoForm.hooks({
 
         // Add current user as Organization manager & creater
         organization.managerIds = [userId];
+        organization.emailVerification = [];
+        // Set default value for the owner of organization
+        organization.emailVerification.push({
```
Comment (marla-singer) [inline apinf_packages/organizations/client/add/autoform.js]: What about:
```js
organization.emailVerification = [{....}];
```
?

File: apinf_packages/organizations/client/managers/list/list.html [hunk 2]
```diff
@@ -9,14 +9,18 @@
...
-        {{# if hasMultipleManagers }}
-          <button
-            id="remove-organization-manager"
-            class="btn btn-danger btn-xs pull-right">
-            <i class="fa fa-trash-o fa-lg" aria-hidden="true"></i>
-          </button>
-        {{/ if }}
+        <div class="pull-right">
+          {{# unless emailVerified }}
+            <span class="label label-default">Unverified</span>
```
Comment (marla-singer) [inline apinf_packages/organizations/client/managers/list/list.html]: 1. i18n tag for this word is missed
2. Nested element should have a new line

File: apinf_packages/organizations/client/managers/list/list.js [hunk 3]
```diff
@@ -25,10 +25,27 @@ Template.organizationManagersList.helpers({
...
     // flatten structure of users within organization managers array
     organizationManagers = _.map(organizationManagers, (user) => {
+      // Construct empty object
+      let verificationData = {};
+      // Get verified status from organization data
+      if (organization.emailVerification) {
+        // Get email verification status
+        const result = _.find(organization.emailVerification, { managerIds: user._id });
+        // In case of existing organizations
+        if (typeof result === 'undefined') {
```
Comment (marla-singer) [inline apinf_packages/organizations/client/managers/list/list.js]: Make invert of this if-else & simplify
```js
// Construct empty object
let verificationData = {};
const result = _.find(organization.emailVerification, { managerIds: user._id });
if (result) {
  varidicationData = result
} else {  
verificationData.verified = organization.createdBy === user._id;
}
```
If `organization.emailVerification` is `undefined` then `result` variable is `undefined` as well
**Remake**

File: apinf_packages/organizations/server/methods.js [hunk 4]
```diff
@@ -243,4 +279,83 @@ Meteor.methods({
...
+        { $set: {
+            'emailVerification.$.verified': true,
+          },
+        }
+      );
+
+      return resp;
+    } else {
+      // Throw Authentication error for client
+      throw new Meteor.Error(`Email verification failed. Authentication token doesn't exists in db.`);
```
Comment (matleppa) [inline apinf_packages/organizations/server/methods.js]: exists --> exist

**Comment by matleppa** [comment]
@deligence-dharmendra I agree with @Nazarah about User getting redirected to  Organization page while verificating.

**Pull request merged**